XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -524,11 +524,19 @@ private void Server_PropertyChanged(object sender, PropertyChangedEventArgs e)
                 UpdateOpenSSHConsoleButtonState();
             }
 
-            if (source.IsControlDomainZero && e.PropertyName == "name_label")
+            if (source.is_control_domain && e.PropertyName == "name_label")
             {
-                HostLabel.Text = string.Format(Messages.CONSOLE_HOST, source.AffinityServerString);
+                bool isDom0 = source.IsControlDomainZero;
+
+                if (isDom0)

```

Comment:
I think the HostLabel.Text needs to be updated for the other control domain as well